samples/quota/manage_create_or_update.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+from azure.identity import DefaultAzureCredential
+from azure.mgmt.quota import AzureQuotaExtensionAPI
+
+credentials = DefaultAzureCredential()
+subscription = os.getenv('AZURE_SUBSCRIPTION_ID')
+
+quota_client = AzureQuotaExtensionAPI(credentials, subscription)
+
+GROUP_NAME = 'zb_test'
+
+result = quota_client.quota.begin_create_or_update(
+    resource_name=GROUP_NAME,
+    scope='/subscriptions/' + subscription + 'providers/Microsoft.Compute/location/easrus',
+    create_quota_request={
+        "properties": {
+            "limit": 200,
+            "name": {
+                "value": GROUP_NAME
+            }
+        }
+    }
+)
```

Comment:
@Wzb123456789  
(1) please add requirements.txt and readme like other samples
(2) please make sure the test can run directly with least dependency. For example, we can add code to create resource group so that users don't need to create it manually